### DISCUSSION TITLE
TVFix

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#802] [Q&A]
Can the channels from this site: http://live.tvfix.org possibly be added? Direct links from that site used to work but not anymore. I wonder if there's a workaround for those to work?

**Comment by ztreamhd** [comment]
Yeah, you can make these work. The original links (example: ``http://live.tvfix.org/hls/{example}.m3u8``) are now containers for the actual M3U8 links. Building a script that could get a response from the original link by using cURL would totally work, since it'd return the playable stream file.

**Comment by thedarkdragon11** [comment]
Still no workaround for this? Tried using cURL using proper headers, while it can get the m3u8 contents, it doesn't play as it is supposed to.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
